docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -21,15 +21,20 @@ APM & {fleet} node. This node is required to use {fleet-server}.
 image::images/fleet-server-hosted-container.png[{fleet-server} hosted agent]
 ====
 
+NOTE: When using our hosted {ess}, it's recommended that you use our hosted
+version of {fleet-server}. However, you can choose to deploy and self-manage
+{fleet-server}s.
+
 // end::ess[]
 
 // tag::self-managed[]
 
-To deploy {fleet-server} on a self-managed cluster, install an {agent} and
-enroll it in the default {fleet-server} policy. The policy is created
-automatically when you run {fleet} for the first time.
+//QUESTION: Is it still recommended that users configure the Fleet Settings up
+//front, or does the Fleet Server host setting in the in-product steps make
+//that unnecessary?
```

Comment:
When the user adds a new fleet server they need to add the host to the list of hosts in fleet settings